### ISSUE TITLE
AttributeError: module 'seal' has no attribute 'EncryptionContext'

### ISSUE THREAD

**Issue by Haonan009** [Huelse/SEAL-Python#109]
**Type**
Import seal in my project.

**Descripe**
I have run files in "examples" successfully, but failed in my project though .pyd file has been added to the directory.

![image](https://github.com/Huelse/SEAL-Python/assets/94544256/eef55b05-15b1-45b0-90eb-5f423895b0f1)

**Comment by Huelse** [comment]
1. Whats your python version in the IDE?
2. `print(dir(seal))` to check the seal lib's modules.
3. Check the installed packages to make sure there is no module named 'seal'

**Comment by Haonan009** [comment]
1. It's python3.7.
2. listed ['BatchEncoder', 'CKKSEncoder', 'Ciphertext', 'CoeffModulus', 'ContextData', 'Decryptor', 'EncryptionParameterQualifiers', 'EncryptionParameters', 'Encryptor', 'Evaluator', 'GaloisKeys', 'KSwitchKeys', 'KeyGenerator', 'Modulus', 'PlainModulus', 'Plaintext', 'PublicKey', 'RelinKeys', 'SEALContext', 'SecretKey', 'VectorDouble', 'VectorInt', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__spec__', '__version__', 'error_type', 'scheme_type', 'sec_level_type']
It turns out some modules which I used were not in the dir(seal).
3. I've pip uninstall seal in my anaconda env.
Was it  because I called some modules which didn't exist in the seal directory?

**Comment by Haonan009** [comment]
> 1. Whats your python version in the IDE?
> 2. `print(dir(seal))` to check the seal lib's modules.
> 3. Check the installed packages to make sure there is no module named 'seal'
1. It's python3.7.
2. listed ['BatchEncoder', 'CKKSEncoder', 'Ciphertext', 'CoeffModulus', 'ContextData', 'Decryptor', 'EncryptionParameterQualifiers', 'EncryptionParameters', 'Encryptor', 'Evaluator', 'GaloisKeys', 'KSwitchKeys', 'KeyGenerator', 'Modulus', 'PlainModulus', 'Plaintext', 'PublicKey', 'RelinKeys', 'SEALContext', 'SecretKey', 'VectorDouble', 'VectorInt', 'doc', 'file', 'loader', 'name', 'package', 'spec', 'version', 'error_type', 'scheme_type', 'sec_level_type']
It turns out some modules which I used were not in the dir(seal).
3. I've pip uninstall seal in my anaconda env.
Was it because I called some modules which didn't exist in the seal directory?
